webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -152,7 +152,6 @@ const ExperimentDetailsHeader: React.FC<Props> = ({
   const handleDescriptionUpdate = useCallback(async (newValue: string) => {
     try {
       await patchExperiment({ body: { description: newValue }, experimentId: experiment.id });
-      await fetchExperimentDetails();
```

Comment:
Removed this additional fetching for two reasons.

1. the main reason it is there is to update the state of the experiment, which is ok, but, since it's an atomic change already reflected in the UI, the UI itself already is reflecting the optimistic rendering of the outcome, making this request avoidable.
2. we're wrapping this change in a `try... catch`, which means that, if it goes bad, the catch would alert the user that something didn't work.
3. there are more of this code in the component, but, it felt kinda unsafe to remove for now.